functions/src/util.ts

```diff
@@ -40,3 +43,30 @@ export const getObjectSizeInBytes = (object: [] | {}) => {
   const recordBuffer = Buffer.from(JSON.stringify(object));
   return recordBuffer.byteLength;
 }
+
+
+export const getFields = (config) => config.fields ? config.fields.split(',') : [];
```

Comment:
since config is also imported, should this still be an argument?

Reply:
I debated this.  Since it was a Util function, I thought it made sense to pass in the config object.